### PR TITLE
add ubuntu 14 instructions and MSSQL driver

### PR DESCRIPTION
**Pull request by lenisha**
@maryamariyan , this change describes the process to install libraries unixODBC and SQLite/MSSQL driver for ubuntu 14

### CODE REVIEW

File: src/System.Data.Odbc/src/DatabaseSetupInstructions.md [hunk 1]
```diff
@@ -23,7 +23,48 @@ Get the tag name from https://hub.docker.com/r/microsoft/dotnet-buildtools-prere
...
+- `sudo nano /etc/odbcinst.ini`
+
+```
+[ODBC Driver 17 for SQL Server]
+Description=Microsoft ODBC Driver 17 for SQL Server
+Driver=/opt/microsoft/msodbcsql17/lib64/libmsodbcsql-17.2.so.0.1
+UsageCount=1
+```
+
+To use Microsoft driver, here is sample `odbc.ini`
```
Comment (danmosemsft) [inline src/System.Data.Odbc/src/DatabaseSetupInstructions.md]: "To use the Microsoft driver, here is a sample `odbc.ini`"

**Comment by v-chojas** [comment]
FYI. unixODBC 2.3.**7** is now available and fixes many bugs. http://www.unixodbc.org/

**Comment by David-Engel** [comment]
This document could use more improvement to bring it up-to-date (more recent OS versions, newer unixODBC version). That said, these changes do bring it forward, matching the current MS ODBC SQL driver instructions and I think they are acceptable.

**Comment by maryamariyan** [comment]
Thanks @v-chojas and @David-Engel for your feedback.

If you would like please go ahead and make the changes as you see fit in a separate PR.

**Comment by karelz** [comment]
@lenisha do you plan to make the suggested change? We can then merge it!
Thanks!

**Comment by karelz** [comment]
Thanks for your PR @lenisha!

**Pull request merged**